lib/common/remote/ipc-messages.ts

```diff
@@ -0,0 +1,19 @@
+export const enum IPC_MESSAGES {
```

Comment:
Please also apply this change to https://github.com/electron/remote.

Reply:
https://github.com/electron/remote/pull/10